installer/InstallBuilder/linuxrpm.py

```diff
@@ -133,7 +134,8 @@ def GenerateSpecFile(self):
 
         specfile.close()
 
-
+        if self.variables['PFDISTRO'] == 'ULINUX' and self.variables['PFARCH'] == 'ppc':
```

Comment:
I think ULINUX is normally only defined if --enable-ulinux is used to configure. Not sure on that, though.

If you build for PPC without --enable-ulinux (because you should not), is ULINIX still defined? If not, then this AND condition would never be true.

Just check this. If it is defined, I'm good. If not, please fix and I'm still good.